dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
If this project does not contain code from another Apache-licensed project, we should use the shorter OpenSearch header

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```